velox/functions/sparksql/tests/RegexFunctionsTest.cpp

```diff
@@ -34,6 +35,10 @@ class RegexFunctionsTest : public test::SparkFunctionBaseTest {
   void SetUp() override {
     SparkFunctionBaseTest::SetUp();
     registerRegexpReplace("");
```

Comment:
You can remove this line now

Reply:
Hi, it's unclear to me which line you are talking about. Are you referring to the GTestUtils.h include? Or the comment and options_.parseIntegerAsBigint?

Reply:
It's this line `registerRegexpReplace("");`

Reply:
Ah, good catch. Removing, thank you!